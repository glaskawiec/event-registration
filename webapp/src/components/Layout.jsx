import React from 'react';
import PropTypes from 'prop-types';
import { Container, Row, Col } from 'react-bootstrap';

const Layout = (props) => (
    <>
        <Container fluid className="center">
            <Row className="d-flex justify-content-center">
                <Col xl={4} lg={5} md={5} sm={7}>
                    <h1 style={{ margin: 30, textAlign: 'center' }}>
                        Event Registration
                    </h1>
                    {props.children}
                </Col>
            </Row>
        </Container>
    </>
)

Layout.propTypes = {
    children: PropTypes.element.isRequired
};

export default Layout;