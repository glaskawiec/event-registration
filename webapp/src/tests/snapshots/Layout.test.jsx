import React from 'react';
import renderer from 'react-test-renderer';
import Layout from '../../components/Layout';

test('Layout snapshot', () => {
    const tree = renderer
        .create(
            <Layout>
                <div>
                    Snapshot
                </div>
            </Layout>
        )
        .toJSON();

    expect(tree).toMatchSnapshot();
});