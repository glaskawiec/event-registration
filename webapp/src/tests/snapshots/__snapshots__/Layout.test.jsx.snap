// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout snapshot 1`] = `
<div
  className="center container-fluid"
>
  <div
    className="d-flex justify-content-center row"
  >
    <div
      className="col-xl-4 col-lg-5 col-md-5 col-sm-7"
    >
      <h1
        style={
          Object {
            "margin": 30,
            "textAlign": "center",
          }
        }
      >
        Event Registration
      </h1>
      <div>
        Snapshot
      </div>
    </div>
  </div>
</div>
`;
